bears/vcs/git/GitRevertInspectBear.py

```diff
@@ -0,0 +1,148 @@
+            minimum_similarity_ratio: float = 0.8,
+            **kwargs):
+        """
+        Inspect the HEAD commit to check if it is a
+        git revert commit and return result accordingly.
+
+        :param allow_git_revert_commit:     Whether revert commit
+                                            is allowed or not.
+        :param minimum_similarity_ratio:    Minimum similarity ratio
+                                            required by files in revert
+                                            commit.
+        """
+        for result in dependency_results[GitCommitMetadataBear.name]:
+
+            m = self.GIT_REVERT_COMMIT_RE.match(result.raw_commit_message)
+            if not m:
+                return
+
+            if not allow_git_revert_commit:
+                yield Result(self, 'Revert commit is not allowed.')
+                return
+
+            reverted_commit_sha = m.group(1)
+            get_files_command = ('git show --pretty="" --name-status %s' %
+                                 reverted_commit_sha)
+
+            all_files = run_shell_command(get_files_command)[0]
+            files_modified_by_reverted_commit = all_files.split('\n')
+
+            reverted_commit_modified_files_list = []
+            reverted_commit_added_files_list = []
+            reverted_commit_deleted_files_list = []
+
+            for line in files_modified_by_reverted_commit:
+                pos = line.find('\t')
+                change = line[:pos]
+                if change == 'M':
+                    reverted_commit_modified_files_list.append(line[pos+1:])
+                elif change == 'A':
+                    reverted_commit_added_files_list.append(line[pos+1:])
+                elif change == 'D':
+                    reverted_commit_deleted_files_list.append(line[pos+1:])
+
+            for file_path in result.added_files:
+                if file_path not in reverted_commit_deleted_files_list:
+                    yield Result(self, 'Revert commit has an added '
+                                       'file %s that is not in reverted '
+                                       'commit.' % file_path)
+
+            for file_path in result.deleted_files:
+                if file_path not in reverted_commit_added_files_list:
+                    yield Result(self, 'Revert commit has a deleted '
+                                       'file %s that is not in reverted '
+                                       'commit.' % file_path)
+
+            for file_path in result.modified_files:
+                if file_path in reverted_commit_modified_files_list:
+                    yield from self._check_modified_file_similarity(
+                        file_path, reverted_commit_sha,
+                        minimum_similarity_ratio)
+
+                else:
+                    yield Result(self, 'Revert commit has a modified '
+                                 'file %s that is not in reverted '
```

Comment:
`... not in the reverted ...`

Reply:
This string should be a constant, with substitution variable for "added", "deleted" & "modified"